chainerio/filesystems/hdfs.py

```diff
@@ -67,11 +68,12 @@ def _create_connection(self):
         if None is self.connection:
             logger.debug('creating connection')
 
-            if None is not self.keytab_path:
-                self.ticket = KrbTicket.init(
-                    self.username,
-                    self.keytab_path)
-                self.ticket.updater_start()
+            # Updater automatically let kinit take ``KRB5_KTNAME``
+            # variable. If /etc/krb5.keytab doesn't exist, krbticket
+            # tries to update the ticket with ``kinit -R`` as much as
+            # possible.
+            self.ticket = KrbTicket.init(self.username)
```

Comment:
Is the update needed when we do not have the keytab?

Reply:
Please read the code comment above

Reply:
I meant do we have to update the ticket even if we do not have the keytab?

Reply:
Yes. Ticket expiration date can be renewable until renewable date by `kinit -R` and the former is usually earlier than latter, like hours or days.

Reply:
I see